source/css/sass/_normalize.scss

```diff
@@ -94,6 +94,18 @@ small {
   @include small-spacing();
 }
 
+sup {
+  @include text-scripts;
```

Comment:
Wondering if there's a better name for this, although I haven't thought of one yet. @thewilkybarkid this mixin describes the common parts of the superscript & subscript sass, any ideas for a name?

Reply:
Might be better to do something like:

```scss
sup {
  @include font-variant-position(sup);
}

@mixin font-variant-position($position) {
  // fallback css
  @supports (font-variant-position: #{$position}) {
    // happy days
  }
}
```

?

Reply:
@thewilkybarkid nice one. Let's do that @discodavey.